config/nuttx/stm32f4dis/patches/0001-Only-call-up_netinitialize-if-network-is-configured.patch

```diff
@@ -0,0 +1,25 @@
+From 662e30417dae4bae722a3408deae930a7917d19d Mon Sep 17 00:00:00 2001
+From: Tomasz Wozniak <t.wozniak@samsung.com>
```

Comment:
I don't want your name to remain here.

Reply:
Why?
In case of this file it no longer matters, since I removed the patch.